Dockerfile

```diff
@@ -7,11 +7,13 @@
 # so we do a multistage build where we generate the requirements.txt in our first stage,
 # and consume it in our final stage.
 # TODO: This slows down the build somewhat, look at alternatives.
-FROM python:3.6
+FROM python:3.8-buster
 # Install pipenv.
 RUN python -m pip install pipenv
 # Copy Pipfile and Pipfile.lock.
 COPY Pipfile* /tmp/
+# TODO: change this, it's totally wrong - it means the Pipfile.lock is ignored, which kind of defeats the
```

Comment:
do we still need these TODOs in this file?

Reply:
Yep - pipenv lock --requirements is not a good way to generate the requirements.txt